### ISSUE TITLE
[Doc]Wrapper configuration has missed some properties

### ISSUE THREAD

**Issue by nilasini** [wso2/product-is#3889]
The documentation \[1] has missed the following property in the wrapper configuration file. Need to add the following property to \[1].

```
wrapper.java.additional.26 = -Dorg.apache.jasper.compiler.Parser.STRICT_QUOTE_ESCAPING=false
wrapper.java.additional.27 = -Dfile.encoding=UTF8
wrapper.java.additional.28 = -DworkerNode=false
wrapper.java.additional.29 = -Dorg.wso2.ignoreHostnameVerification=true
```

\[1] https://docs.wso2.com/display/IS510/Installing+as+a+Windows+Service

**Comment by ruwanta** [comment]
Also in the same doc, please explain why it is not packed with the product itself.
"Reason is WAJSW is distributed as LGPL and WSO2 is distributed under Apache2. Hence these can not be packed together in a distribution. But end-user or customer can freely combine components with these two licenses, as long as they do not distribute the combined work"

**Comment by sherenewso2** [comment]
Fixed in doc versions 5.2.0 - 5.7.0 [1] and added the reason at the top of the page. 

The following property was different in 5.2.0 and 5.3.0 and had been changed from 5.4.0 onwards. 
Updated the code block in this page [1] accordingly. 
5.3.0 - `wrapper.java.additional.29 = -Dorg.wso2.ignoreHostnameVerification=true`
5.4.0 onwards - `wrapper.java.additional.29 = -Dhttpclient.hostnameVerifier=DefaultAndLocalhost`

[1] https://docs.wso2.com/display/IS570/Installing+as+a+Windows+Service
